modules/migration-compatibility-guidelines.adoc

```diff
@@ -33,7 +33,7 @@ Legacy {mtc-version} operator: Install manually with the `operator.yml` file.
 This cluster cannot be the control cluster.
 ====
 
-|{mtc-short} {mtc-version}.z
+|{mtc-short} {mtc-version}._z_
```

Comment:
I'm fine with this, since there is elsewhere in this section that italicized it. 

But I'm not positive that that's really the right way to display it. Elsewhere in the docs when we talk about versions with a ".z", we don't italicize them. Or if it's supposed to be a replaceable value, then it's in angle brackets, like \<z>.

Again, just food for thought, nothing to hold up the PR over. Merging!